sklearn/cluster/tests/test_k_means.py

```diff
@@ -542,9 +542,9 @@ def test_predict():
 
 
 def test_score():
-    km1 = KMeans(n_clusters=n_clusters, max_iter=1, random_state=42)
+    km1 = KMeans(n_clusters=n_clusters, max_iter=1, random_state=42, n_init=1)
```

Comment:
Ok, I figured why this is necessary. Out of the 10 restarts, we not pick the one that actually gives the best score, not where the previous iteration gave the best score.
This means in this PR, a different random initialization is picked than in master, which gives better results.